languages/cpp/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,43 @@
+C++ is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by letting the C++ compiler infer its type based on the assigned value (known as _type inference_). Therefore, the following two variable definitions are equivalent:
```

Comment:
Should there be mention of declaration, as well as definition?
Roughly similar like:
    "... creating a name is referred to as declaring a variable, assigning a value to a declared variable is referred to as defining a variable. Declaration and definition can be performed separately or as part of a single statement ..."

Reply:
You're right, I like how you defined it, I'll probably use it and. I should add that variable declared and not initialized are not exactly valid.